packages/metadata-service/src/readers/chitai-gorod.service/index.ts

```diff
@@ -0,0 +1,73 @@
+import { BadRequestException, Injectable } from '@nestjs/common';
+import axios from 'axios';
+import { Prisma } from '@prisma/client';
+import { Bundle } from '../ozby.service';
+
+@Injectable()
+export class ChitaiGorodService {
+  private READER_ID = 'Сhitai-gorod';
+  private readonly rootUrlSearch =
+    'https://search-v2.chitai-gorod.ru/api/v3/search/';
+  private readonly rootUrlList =
+    'https://webapi.chitai-gorod.ru/web/goods/extension/list/';
+
+  async readData(identifier: string) {
+    try {
+      const resultOfId = await axios.post(this.rootUrlSearch, {
+        index: 'goods',
+        query: identifier,
+        type: 'common',
+        from: 0,
+        per_page: 24,
+        'filters[available]': false,
+      });
+      const ids = resultOfId.data.ids[0];
+      const infoOfBook = await axios.post(this.rootUrlList, {
+        token: 123,
+        action: 'read',
+        data: [ids],
+      });
+      return infoOfBook.data.result[ids];
+    } catch (e) {
+      throw new BadRequestException(e.message);
+    }
+  }
+
+  parseData(result): Bundle {
+    if (!result) return null;
+    const material: Prisma.MaterialCreateInput = {
+      title: result.name,
+      cover: `https://img-gorod.ru${result.image_url}`,
+      yearPublishedAt: result.year,
+      monthPublishedAt: 0,
+      description: result.detail_text,
+      meta: {
+        price: result.price,
+        sku: result.book_id,
+        taxonomy: result.section_names,
+      },
+    };
+
+    const authors: Array<Prisma.AuthorCreateInput> = Array.isArray(
+      result.author
+    )
+      ? result.author.map((author) => ({
+          name: author,
+          referenceId: this.READER_ID,
```

Comment:
This should be a unique id provided by chitai-gorod, not just a const. This `referenceId` is used keep a reference to provider's entity and also for unique identification.